lib/public-stats.js

```diff
@@ -241,10 +241,8 @@ const updateRetrievalTimings = async (pgClient, committees) => {
     const { minerId } = c.retrievalTask
     const ttfbMeasurments = []
     for (const m of c.measurements) {
-      if (m.taskingEvaluation !== 'OK' || m.consensusEvaluation !== 'MAJORITY_RESULT') continue
-      // FIXME: assert first_byte_at and start_at during preprocessing
-      // See https://github.com/filecoin-station/spark-evaluate/issues/447
-      if (!m.first_byte_at || !m.start_at || m.start_at > m.first_byte_at) continue
+      if (m.retrievalResult !== 'OK' || m.taskingEvaluation !== 'OK' || m.consensusEvaluation !== 'MAJORITY_RESULT') continue
+
       const ttfbMeasurment = m.first_byte_at - m.start_at
       if (isNaN(ttfbMeasurment)) continue
```

Comment:
I still left this as a runtime check, although we should never trigger this statement

Reply:
Let's remove it, that code before make this unnecessary